src/Sulu/Bundle/AdminBundle/Resources/js/components/Form/grid.scss

```diff
@@ -12,8 +11,9 @@
     margin-bottom: -$fieldMarginBottom;
 
     &:first-child {
-        > .divider-container {
-            margin-top: -$dividerTopMargin;
+        /* the first section divider should not have a margin top */
```

Comment:
should we target the `2.2` branch for this change?

Reply:
@nnatter the 2.2 does not have sections inside blocks.

Reply:
alright 🙂